README.md

```diff
@@ -1,30 +1,34 @@
 # Timed Application
 
 ## Introduction
+This chart runs a [timed](https://github.com/adfinis-sygroup/timed-frontend) deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
-This chart bootstraps a Timed application deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+Current chart version is `0.2.2`
```

Comment:
```suggestion
```
I'm pretty sure adding this to the readme will only be a pain because we update it all the time.

Maybe linking the Releases tab and telling folks to look there makes sense.

Reply:
makes sense

Reply:
Now that I realize that this is being autogenerated using helm-docs... does this change persist?

Reply:
@MrStrix Sorry about the confusion. You should either revert the change I requested here or a a template to ensure that it persists running helm-docs again.

Reply:
I don't know if we should use helm-docs regularly. I intended to use it for initial generation only, further additions to the table can easily be done manually imo. Otherwise I'd suggest to use a specific template and pipeline to autogenerate the README.md

Reply:
I don't seem to get how github actions work... Job has completed but where does it put the generated README?

Reply:
Ah got it, it has to be created with the pre-commit hook, not by the action

Reply:
It only complains if the README is outdated, it doesn't automate creating/updating the file.